less/home-page.less

```diff
@@ -6,10 +6,16 @@
 
     h2 {
       padding: 15px;
+      @media @mobile{
+        padding: 15px 0;
+      }
     }
 
     p {
       padding:0 15px;
+      @media @mobile{
+        padding: 0 0;
+      }
     }
```

Comment:
One way you could simplify your code is to set padding up and down but not left and right. Use margins on the entire home container so that it is nice and centered on mobile and desktop view.